src/test/java/picard/analysis/artifacts/TransitionTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2015 Daniel Gómez-Sánchez
```

Comment:
2017

Reply:
Sorry, this was automatically generated by my IDE (I should update it eventually); nevertheless, I do not want to add the LICENSE notice to the file, so I removed it completely.